### ISSUE TITLE
Installing cabal2nix

### ISSUE THREAD

**Issue by tekul** [basvandijk/nix-workshop#1]
I'm just running through the exercises from the workshop and hit an issue where `cabal2nix` wasn't found at [this line](https://github.com/basvandijk/nix-workshop/blob/712f4942226fcad165f905b5d5784f6e2fcf4d4c/02-nix-based-development/run.sh#L16). I installed nix on my machine directly rather than using the docker image.

After running `nix-env -i cabal2nix` as described in the manual (not sure if that's the best option here), I was able to proceed.

**Comment by peti** [comment]
Yes, this is a good point. The docker image has cabal2nix pre-installed (as well as ghc, cabal-install, stack, etc.), but users running the examples in a different environment won't have them. I'll take a shot at running the `run.sh` scripts in an empty env soon and make sure they are self-contained.

**Comment by tekul** [comment]
Thanks. Perhaps even just a note to that effect would be enough. Most Haskell programmers will probably be familiar with cabal and stack and know how to install them if they haven't already. cabal2nix is likely to be new though.
